org.musicbrainz.Picard.json

```diff
@@ -153,11 +160,11 @@
             "name": "python-pkgconfig",
             "buildsystem": "simple",
             "build-commands": [
-                "python2 setup.py install --prefix=/app"
+                "pip3 install --no-deps --prefix=/app pkgconfig-1.3.1.tar.gz"
```

Comment:
Please use `pip3 install --no-deps --prefix=/app .` as I had mentioned, keeping the `archive` source type.

Reply:
Oh OK ! I missed the final dot in your comment. ^^